src/backend/core/api/serializers.py

```diff
@@ -165,7 +164,6 @@ class Meta:
         ]
         read_only_fields = [
             "id",
-            "abilities",
```

Comment:
It will break the frontend if you remove it. We use the abilities in the data grid, to know if the current user can delete or not a document.
We use it as well to compute the current role of the user on the document (not sure we still do it with the new design) though.